plugins/hls-pragmas-plugin/test/testdata/AddLanguagePragma.expected

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TupleSections #-}
+
+module NeedsLanguagePragms where
+
+tupleSection = (1,) <$> Just 2
+
+{-# INLINE addOne #-}
+addOne :: Int -> Int
+addOne x = x + 1
```

Comment:
Is this file generated? Why doesn't this name contain ".hs" extension?

Reply:
hi, no I tried the test manually first and then added it to the test suite but I must have left off the .hs, thanks for pointing it out.